### PR TITLE
Assortment of fixes

### PR DESCRIPTION
**Pull request by bd82**
* Fixed MIN_MAX keywords parent.
* Consistent naming for grammar rules.
* Removed a ".only" mocha suffix and added an eslint rule to detect such
  errors.

### CODE REVIEW

**Comment by bd82** [comment]
Important to merge this quick as only one test is running now due to mistake in previous PR. 😢

File: package.json [hunk 1]
```diff
@@ -51,6 +51,7 @@
     "eslint": "4.8.0",
     "eslint-config-airbnb-base": "12.0.2",
     "eslint-plugin-import": "2.7.0",
+    "eslint-plugin-mocha": "^4.11.0",
```
Comment (deepu105) [inline package.json]: can you use a fixed version for this in the next PR? i'll merge this now

Comment (MathieuAA) [reply package.json]: @bd82 don't forget to run `yarn` to regenerate the yarn.lock file!

Comment (bd82) [reply package.json]: Opps, I used npm install instead of yarn add :(

**Pull request merged**